src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -135,23 +106,34 @@ const WorkspaceContainer = ({ namespace, name, breadcrumbs, topBarContent, title
     ]),
     showTabBar && h(WorkspaceTabs, { namespace, name, activeTab, refresh, workspace, deletingWorkspace, setDeletingWorkspace, cloningWorkspace, setCloningWorkspace, sharingWorkspace, setSharingWorkspace }),
     div({ role: 'main', style: Style.elements.pageContentContainer },
-      // When we switch this over to all tabs, ensure other workspace tabs look the same when inside these divs
-      (isAnalysisTabVisible() && activeTab === 'analyses' ?
+
+      // TODO: When we switch this over to all tabs, ensure other workspace tabs look the same when inside these divs
+      (isAnalysisTabVisible() && (activeTab === 'analyses' || activeTab === undefined) ?
         [div({ style: { flex: 1, display: 'flex' } }, [
-          div({ style: { flex: 1 } }, [
+          div({ style: { flex: 1, display: 'flex', flexDirection: 'column' } }, [
             children
           ]),
           workspace && h(ContextBar, {
-            setDeletingWorkspace,
-            setCloningWorkspace,
-            setSharingWorkspace,
-            isOwner,
-            canShare,
+            workspace, isOwner, canShare, setDeletingWorkspace, setCloningWorkspace, setSharingWorkspace,
             canCompute: !!(workspace?.canCompute || runtimes?.length),
-            workspace, refreshApps, refreshRuntimes,
-            runtimes, persistentDisks, galaxyDataDisks, apps
+            apps, galaxyDataDisks, refreshApps,
+            runtimes, persistentDisks, refreshRuntimes
```

Comment:
Not your fault, but we need your help cleaning things up. `isOwner`, `canShare` , and `canCompute` can be derived from the other data passed to `ContextBar` here, so should be removed from here and calculated there instead.